### PR TITLE
Add test for confirming desired elo funcionality

### PR DESCRIPTION
**Pull request by timiimit**

### CODE REVIEW

**Review by Saibamen — CHANGES_REQUESTED** [review]

File: .Tests/XUnit.Tests/EloTest.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		}
+
+		public override string ToString()
+		{
+			return $"Actual Skill: {ApproxSkillRating}], Current Rating: {Rating}";
+		}
+	}
+
+	[Theory]
+	[InlineData(100, 5, 1000)]
```
[the remark this answers is not in the document]
Comment (timiimit) [reply .Tests/XUnit.Tests/EloTest.cs]: there's only so many possible tests. cant really think of any edge cases other than 1v1 match.

[the remark this answers is not in the document]
Comment (Saibamen) [reply .Tests/XUnit.Tests/EloTest.cs]: So please add 1-2 more test and 1vs1 match. Otherwise, this test is not a `[Theory]`, but `[Fact]`

[the remark this answers is not in the document]
Comment (timiimit) [reply .Tests/XUnit.Tests/EloTest.cs]: I did add 7 cases in total... including a 1vs1

**Comment by Saibamen** [comment]
Related: #138

**Review by Saibamen — APPROVED** [review]

File: .Tests/XUnit.Tests/EloTest.cs [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+			// update ratings in players
+			for (int j = 0; j < playerCount; j++)
+			{
+				players[j].Rating = shuffledPlayerRatings[shuffledPlayerIndices[j]];
+			}
+		}
+
+		double avg = players.Select(x => x.Rating).Average();
+
+		double expectedAvg = players.Select(x => x.ApproxSkillRating).Average();
```
Comment (Saibamen) [inline .Tests/XUnit.Tests/EloTest.cs:94]:
```suggestion
		double avg = players.Select(x => x.Rating).Average();
		double expectedAvg = players.Select(x => x.ApproxSkillRating).Average();
```
